build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+#
+# Simple "build" script that just copies the app into the build/ directory.
+#
+
+mkdir build/
+rm -rf build/*
+cp -r todos/*.html todos/*.css todos/*.js todos/fontawesome build/
```

Comment:
Think we will need to change all the 'todos' to 'src' here

Reply:
Updated. Thank you!